metrics/src/main/java/io/opentelemetry/example/metrics/DoubleCounterExample.java

```diff
@@ -24,9 +23,8 @@
  */
 public final class DoubleCounterExample {
 
-  private static final OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
   private static final Tracer tracer =
-      openTelemetry.getTracer("io.opentelemetry.example.metrics", "0.13.1");
+      GlobalOpenTelemetry.getTracer("io.opentelemetry.example.metrics", "0.13.1");
```

Comment:
I think this is probably more common usage(?)
```suggestion
      GlobalOpenTelemetry.getTracer("io.opentelemetry.example.metrics");
```